packages/devtools_app/lib/src/memory/memory_snapshot_models.dart

```diff
@@ -46,6 +48,18 @@ class Reference extends TreeNode<Reference> {
 
   Reference.analysis(this.controller, this.name, {this.onExpand, this.onLeaf})
       : isAnalysis = true,
+        isSnapshot = false,
```

Comment:
nit: you could simplify this code by adding a
`Reference._({...})` constructor with optional named parameters with default values.

Reply:
Good idea, I'll add post the PR (added TODO).